src/chronos.erl

```diff
@@ -78,12 +80,11 @@ stop_timer(ServerName, TimerName) ->
 %%%===================================================================
 
 
--spec init(server_name()) -> {'ok', #chronos_state{}} | {'stop', term()} | 'ignore'.
-init(ServerName) ->
-    gproc:reg(proc_name(ServerName)),
+-spec init([]) -> {'ok', state()} | {'stop', term()} | 'ignore'.
+init(_Args) ->
     {ok, #chronos_state{}}.
 
--spec handle_call(term(), term(), #chronos_state{}) ->
+-spec handle_call(term(), term(), state()) ->
                          {'reply', 'ok' | {'error', term()} , #chronos_state{}} |
```

Comment:
Line 88 is too long:                          {'reply', 'ok' | {'error', term()} , #chronos_state{}} |.